### PR TITLE
Cleanup raster images (Fixes #1961)

### LINKED ISSUE #1961: Clean up project from raster images
[comment]: # (Please replace ... with your information. Remove < and >)

Most images have been moved to vector https://github.com/status-im/status-react/tree/develop/resources/icons

We need to clean up ios and Andorid (https://github.com/status-im/status-react/tree/develop/android/app/src/main/res) projects from raster images

### PR DESCRIPTION
**Pull request by monokh**
### Summary:

Cleaning up raster images in the project which are unused. 

### Review notes:

TODO: 
- [x] Remove raster images which are now used from vector-icons
- [x] Remove any other raster images

1. Is there a definitive way to check which raster images are unused? So far have just been looking it up in code. For example, would builds fail upon missing resources?
2. Are all raster images that are unused to be removed? Even if they don't have equivalant `vector-icon`s ?

Fixes #1961

### CODE REVIEW

**Comment by jeluard** [comment]
@monokh Thanks for working on this!

>Is there a definitive way to check which raster images are unused? So far have just been looking it up in code. For example, would builds fail upon missing resources?

I don't think so no unfortunately.

>Are all raster images that are unused to be removed? Even if they don't have equivalant vector-icons ?

Please go ahead and remove all of them.

**Comment by monokh** [comment]
I've removed any images i could positively identify weren't used.
I'd like to try it out with the changes but I'm having some issues building a development build at the moment. 
I'll let you know when I've tried it out but in the meantime feel free to merge/check it on your side.

File: android/app/build.gradle [hunk 1]
```diff
@@ -234,7 +234,7 @@ task hemroidBuild(type: Exec) {
     def rootDir = project.rootDir
     def localProperties = new File(rootDir, "local.properties")
 
-    def ndkDir = "$System.env.ANDROID_NDK_HOME"
+    def ndkDir = "/Users/cappuccino/Library/Android/sdk/ndk-bundle"
```
Comment (jeluard) [inline android/app/build.gradle]: This probably should not be there.

**Review by jeluard — COMMENTED** [review]
How can we help you build locally?

File: android/app/build.gradle (hunk 1 above)
Comment (monokh) [reply android/app/build.gradle]: Correct, these are a sign of my battles with getting a dev build 😄

**Comment by monokh** [comment]
@jeluard I got some pointer from the riot chat. Trying again just now 😃  hopefully i'm on the right track here

**Comment by monokh** [comment]
@jeluard managed to get a build going! 😃  Wasn't successful with the dev build but managed to build for release with the changes. 

Tried every accessible view doing a comparison between this branch's release build vs develop and found 3 icons that I had deleted but was actually used. Re-added them with  https://github.com/status-im/status-react/pull/2748/commits/74f38daa558c35bba39e4021e2e7df5980b71e98 & https://github.com/status-im/status-react/pull/2748/commits/35666d46c6404429be5dcd4a04d8f75f6a31f17d. Testing after this, the icons appear again.

Let me know if anything else :slightly_smiling_face:

**Comment by flexsurfer** [comment]
great job @monokh , thanks for your contribution, let's test it, moved issue to testing :)

**Review by jeluard — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Comment by churik** [comment]
Checked:
*install 
- all default avatars: 1-1 Chat, group Chat, Public Chat,
- icons on discover screen
- icons on contacts screen
- messaging in 1-1, Group chat

*upgrade
- with 1-1 chat, group chat 
- users with default profile picture and profile picture which is set from Gallery.
in IOS works correct, on android upgrade check is blocked by #2762

**Pull request merged**